host/frontend/operator/tags

```diff
@@ -0,0 +1,8 @@
+Mmain.go	main.go	15
```

Comment:
wondering what's this file for? is it vim tags?

Reply:
yes, removed it.